cognite/neat/graph/extractors/_iodd.py

```diff
@@ -127,9 +131,9 @@ def _process_data_collection2triples(
             pc_root, "ProcessDataIn", ignore_namespace=True, include_nested_children=True
         ):
             for process_data_element in process_data_in:
-                if id := process_data_element.attrib.get("id"):
+                if process_data_element.attrib.get("id"):
                     device_id_str = remove_namespace_from_uri(device_id)
-                    process_data_in_id = namespace[f"{device_id_str}.{id}"]
+                    process_data_in_id = namespace[f"{device_id_str}.{P_DATA_IN_ID_OVERRIDE}"]
```

Comment:
There is only one process data in per equipment/IODD. So `process_data_in_id ` will be unique because of the prefix: device_id_str

Reference for ProcessDataIn is here: https://ioddfinder.io-link.com/productvariants/search/43793